Resources/Prototypes/Entities/Objects/Power/powercells.yml

```diff
@@ -1,32 +1,73 @@
-﻿- type: entity
-  id: PowerCellSmallBase
+﻿# Power cells
+
+- type: entity
+  id: PowerCellBase
   abstract: true
   parent: BaseItem
   components:
   - type: Physics
     anchored: false
     shapes:
     - !type:PhysShapeAabb
-      bounds: "-0.15,-0.3,0.2,0.3"
+      bounds: "-0.15,-0.3,0.2,0.3" # TODO: these are placeholder values
       layer:
       - Clickable
   - type: PowerCell
-  - type: Appearance
   - type: Sprite
     netsync: false
 
+- type: entity
+  id: PowerCellSmallBase
+  abstract: true
+  parent: PowerCellBase
+  components:
+  - type: PowerCell
+    cellSize: Small
+
+- type: entity
+  id: PowerCellMediumBase
+  abstract: true
+  parent: PowerCellBase
+  components:
+  - type: PowerCell
+    cellSize: Medium
+
+- type: entity
+  id: PowerCellLargeBase
+  abstract: true
+  parent: PowerCellBase
+  components:
+  - type: PowerCell
+    cellSize: Large
+
+- type: entity
+  name: potato battery
+  description: Someone's stuck two nails and some wire in a large potato. Somehow it provides a little charge. You might be able to cram it into an M-sized slot.
+  id: PowerCellMediumPotato
+  parent: PowerCellMediumBase
+  components:
+  - type: Sprite
+    sprite: Objects/Power/Batteries/potato_battery.rsi
+    layers:
+      - state: potato_battery
+  - type: PowerCell
+    maxCharge: 360
+    startingCharge: 360
+    updateVisual: false
+
 - type: entity
   name: small standard power cell
+  description: A rechargeable standardized power cell, size S. This is the cheapest kind you can find.
   id: PowerCellSmallStandard
   parent: PowerCellSmallBase
   components:
   - type: Sprite
-    sprite: Objects/Power/PowerCells/power_cell_small_st.rsi
+    sprite: Objects/Power/Batteries/power_cell_small_st.rsi
```

Comment:
If the folder's being renamed wouldn't you also rename all the other references to PowerCell to Battery?